_SPSS_archived/All_years/02-Lookups/99_extract_NSU_data.R

```diff
@@ -42,4 +42,4 @@ if (file_exists(file_path)) {
 }
 
 nsu_data %>%
-arrow::write_parquet(file_path, compression = "zstd", compression_level = 10)
+  arrow::write_parquet(file_path, compression = "zstd", compression_level = 10)
```

Comment:
The github action keeps styling this file whenever i revert it. This change was made directly on the sept-23-update branch and i had thought this was up to date with it